VCFv4.3.tex

```diff
@@ -503,41 +503,100 @@ \subsection{VCF tag naming conventions}
 
 \section{INFO keys used for structural variants}
```

Comment:
If we're changing the SV fields, can we fix https://github.com/samtools/hts-specs/issues/133 while we're at it? The Number annotations are inconsistent between various SV INFO fields and half of them do not allow multiple SV ALT alleles.

My preference is only specify that only a single symbolic ALT allele per VCF record is allowed.